.bazelrc

```diff
@@ -12,7 +12,7 @@ common:test --test_env=GTEST_INSTALL_FAILURE_SIGNAL_HANDLER=1
 ##
 ##  Custom toolchain.
 ##
-build --crosstool_top=@llvm_toolchain_12_0_0//:toolchain --copt=-D_LIBCPP_ENABLE_NODISCARD
```

Comment:
Why do we not need this `D_LIBCPP_ENABLE_NODISCARD` definition anymore?

Reply:
No idea. I was following suggestions around crosstool_top. I can add it back in.